frontend/src/Components/ActivityModal/ActivityModal.js

```diff
@@ -0,0 +1,54 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { createActivity } from "../../store/activities";
+import SearchPlacesInput from "../SearchPlacesInput/SearchPlacesInput";
+import './ActivityModal.css'
+
+
+const ActivityModal = ({itineraryId}) => {
+    const dispatch = useDispatch();
+    const [title, setTitle] = useState("");
+    const [place, setPlace] = useState(null);
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(createActivity(itineraryId, {
+            title,
+            date: "2023 Aug 03",
```

Comment:
Looks good! Just set the date to `null` here and leave a comment as a reminder to change it later